slam-ws/src/load_waypoints/srv/LoadWaypoint.srv

```diff
@@ -0,0 +1,3 @@
+int32 waypoint_number
```

Comment:
Minor: Let's rename this to `WaypointRequest.srv` since it better represents what this file is!